theme_ipsum/templates/layout.xml

```diff
@@ -0,0 +1,156 @@
+                                            alt="social-media-icon"
+                                            class="social-media-icon"
+                                        />
+                                    </a>
+                                </li>
+                                <!-- Icon -->
+                                <li t-if="website.social_youtube">
+                                    <a
+                                        t-att-href="website.social_youtube"
+                                        target="_blank"
+                                    >
+                                        <img
+                                            src="/theme_ipsum/static/img/icons/youtube.svg"
+                                            alt="social-media-icon"
+                                            class="social-media-icon"
+                                        />
+                                    </a>
+                                </li>
+                                <!-- Icon -->
+                                <li t-if="website.social_github">
+                                    <a
+                                        t-att-href="website.social_github"
+                                        target="_blank"
+                                    >
+                                        <img
+                                            src="/theme_ipsum/static/img/icons/github.svg"
+                                            alt="social-media-icon"
+                                            class="social-media-icon"
+                                        />
+                                    </a>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                <!-- Footer Bottom -->
+                <div class="footer-bottom">
+                    <div class="container">
+                        <p>© Copyright 2020.<span
+                                class="d-none d-xl-inline"
+                            >•</span></p>
+                        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.<span
+                                class="d-none d-xl-inline"
+                            >•</span></p>
+                        <p class="d-none d-lg-flex">
+                            <a href="#">License CC-BY-SA-4.0<span>•</span></a>
+                            <a href="#">Terms &amp; conditions<span>•</span></a>
+                            <a href="#">Privacy policy</a>
+                        </p>
+                    </div>
+                </div>
+            </footer>
+        </xpath>
+    </template>
+    <!-- Add custom toggler icon -->
+    <template id="submenu" inherit_id="website.submenu">
+        <xpath expr="//a[@data-toggle='dropdown']/span" position="after">
+            <t t-if="submenu.classes and 'btn' in submenu.classes">
```

Comment:
It seems this `classes` attribute is defined in the `website_menu_class` module, but it's not yet a dependency of the Ipsum theme, and I'm not sure if it's even in OCA yet... so upon installation and usage now you get this error:

![image](https://user-images.githubusercontent.com/1466356/131649857-825cb3be-eecf-4d29-aff2-02885c55c4ee.png)

Reply:
Fixed it